lit-action-solana-decrypt-and-sign/nodejs/src/litAction.js

```diff
@@ -0,0 +1,41 @@
+const {
+  clusterApiUrl,
+  Connection,
+  Keypair,
+  PublicKey,
+  SystemProgram,
+  Transaction,
+} = require("@solana/web3.js");
+const bs58 = require("bs58");
+
+(async () => {
+  const solanaPrivateKey = await Lit.Actions.decryptAndCombine({
+    accessControlConditions,
+    chain: "ethereum",
+    ciphertext,
+    dataToEncryptHash,
+    authSig: null,
+    sessionSigs,
```

Comment:
Why have we provided a sessionSig if authSig is null? It should take the authSig from the one provided while executing the Lit Action

Reply:
Also sessionSigs isn't even a param for this op_code: https://github.com/LIT-Protocol/lit-assets/pull/1167/files#diff-14e850f99e65a75544aa9f9ea5fd6c1f9833fdda0b8e43e930763f467c2170adR303